extensions/interactions/GraphInput/directives/graph-utils.service.ts

```diff
@@ -155,7 +155,7 @@ export class GraphUtilsService {
       return currentValue;
     });
 
-    if (pivot === null) {
+    if (pivot === null || successor === null) {
```

Comment:
Why this change?

Reply:
TypeScript doesn't realize that if `pivot` is null, `successor` must also be null, so it gave a warning in the later lines that `Type 'null' cannot be used as an index type`. I figured this is a better solution than coercing `successor` to be a number because it's a good sanity check.

Reply:
Okay, yep make sense, I didn't realize that if `pivot` is `null` the `successor` also needs to be `null`.